k8s/desiretask_test.go

```diff
@@ -0,0 +1,52 @@
+package k8s_test
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/julz/cube/k8s"
+	"github.com/julz/cube/opi"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	av1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Desiretask", func() {
+	var (
+		client  *kubernetes.Clientset
+		desirer *k8s.TaskDesirer
+	)
+
+	BeforeEach(func() {
+		config, err := clientcmd.BuildConfigFromFlags("", filepath.Join(os.Getenv("HOME"), ".kube", "config"))
+		if err != nil {
+			panic(err.Error())
+		}
+
+		client, err = kubernetes.NewForConfig(config)
+		if err != nil {
+			panic(err.Error())
+		}
+
+		desirer = &k8s.TaskDesirer{Client: client}
+	})
+
+	AfterEach(func() {
+		desirer.DeleteJob("guid")
+	})
+
+	It("Creates deployments for every LRP in the array", func() {
```

Comment:
should tasks really be deployments? I think a task has different semantics (should be only 1, run to completion..)

Reply:
This is a copy paste issue. It should be "It runs a job for every task in the array".